### DISCUSSION TITLE
copyable text param

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1302] [Ideas] [answered]
can we add a `height` param to the `ui.copyable_text()` component?

Example use case:
- text box of certain height
- user enters text and hits Submit
- if the query was successfully, user gets back a "copyable text box" of same height (i.e. `ui.copyable_text()` of same height)

**Comment by mturoci** [comment, marked as answer]
Agree, the `height` attribute might be useful if `multiline=True`, same as `ui.textbox` works today. Tracked in https://github.com/h2oai/wave/issues/1308
